antenna-documentation/scripts/execute_github_site_maven_plugin.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash -e
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+
+BASEDIR=$(readlink -f "$0")
+BASEDIR=$(dirname "$BASEDIR")
```

Comment:
To avoid mutation:
```suggestion
BASEDIR=$(dirname "$(readlink -f "$0")")
```

Better:
```suggestion
BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```